src/garage/torch/algos/loss_function_utils.py

```diff
@@ -1,10 +1,25 @@
-"""loss function utilities."""
+"""Differentiable utils typically used when computing loss functions."""
 import torch
 import torch.nn.functional as F
 
 
 def compute_advantages(discount, gae_lambda, max_len, baselines, rewards):
-    """Calculate advantages."""
+    """
+    Calculate advantages.
+
+    Advantages are a discounted cumulative sum.
```

Comment:
They are much more than this and calculating them using a GPU is fairly complicated. Please see the comprehensive documentation in the TF version for a guide of the level of detail required.

Reply:
@ryanjulian I add few more descriptions to `compute_advantages`
I chekced [this](https://github.com/rlworkgroup/garage/blob/09ec44b4adee0373d6cd7090f79a1e14ac56219c/src/garage/tf/misc/tensor_utils.py#L209) but I think this could be not enough.
What do you think?

Reply:
if the comment you linked still applies to this implementation, feel free to copy and paste it.